aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +252,41 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form guidance
```

Comment:
```suggestion
## Full-page form
```
Can we omit "guidance"? Also, I'm questioning its placement. I feel like it should come before "Exceptions".

Reply:
honestly the whole placement we went back and forth on this page really need some loving or a god to fix the whole format. There is everything everywhere so I made the main section `full page form` then everything related lives within

Reply:
Yeah the page does need some help. The top-half seems to have some structure but then it becomes a bit more free for all hah

Reply:
yeah forsure but also even the top half for example when you look at the dos there are things in there referring to required fields same with the Dont and also another example with required field so would be nice is all of that was in one place

Reply:
but this is a seperate issue ☝️